templates/user/edit-profile.html

```diff
@@ -148,7 +154,7 @@
                                 <td colspan="2"><span class="inline-header">
                                     {% if profile.is_totp_enabled %}
                                         {{ _('Two Factor Authentication is enabled.') }}
-                                        <a href="{{ url('disable_2fa') }}" class="button inline-button">Disable</a>
+				                        <a href="javascript:TwoFactorAuth();" class="button inline-button">Disable</a>
```

Comment:
*gasp* still tabs, and bad indentation, and the disabling page 100% disabled... what are you doing?